src/styles/main.scss

```
@@ -1,3 +1,56 @@
+* {
+  box-sizing: border-box;
+}
+
+
+$days: ('mon':0, 'tue':1, 'wed':2, 'thu':3, 'fri':4, 'sat':5, 'sun':6);
+$cell_width:100px;
+$gap:1px;
+$month_length:(28,29,30,31);
+@mixin calendar_block_cell{
+  width:100px;
+  height:100px;
+  background-color:#eee;
+  border: 1px solid black;
+}
+
+@for $i from 1 through 31 {
+.calendar_block:nth-child(#{$i})::before {
+  content: '#{$i}';
+}
```

Check correct indents
```suggestion
@for $i from 1 through 31 {
  .calendar_block:nth-child(#{$i})::before {
    content: '#{$i}';
  }
```